internal/cloud/awscloud/awscloud.go

```diff
@@ -301,12 +306,13 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, rpmArch str
 	}
 
 	logrus.Infof("[AWS] 📋 Registering AMI from imported snapshot: %s", *snapshotID)
+	uniqueName := fmt.Sprintf("%s-%s", name, uuid.New().String())
 	registerOutput, err := a.ec2.RegisterImage(
 		&ec2.RegisterImageInput{
 			Architecture:       aws.String(ec2Arch),
 			BootMode:           bootMode,
 			VirtualizationType: aws.String("hvm"),
-			Name:               aws.String(name),
+			Name:               aws.String(uniqueName),
```

Comment:
This change is breaking onPrem.
OnPrem needs to have the `ImageName` set in AMI Name (without any UUID appended to it)